recettes/sucrees/tarte_citron_meringuee.md

```diff
@@ -0,0 +1,47 @@
+# Tarte au citron meringuée
+
+---
+
+### Ingrédients
+
+- [ ] 1 (pâte sablée)[./pate_sablee.md]
+
+###### Crème au citron
+
+- [ ] 180g sucre
+- [ ] 2 citrons
+- [ ] 4 oeufs
+- [ ] 20cl crème liquide
+
+###### Meringue
+
+- [ ] 2 oeufs
```

Comment:
plus clair je pense de dire ici : 4 blancs d'oeufs

Reply:
Yeah but that makes things unclear above when we say 2 oeufs for the crème au citron. I think you're right this is better, but I'll change the above as well